tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts

```diff
@@ -0,0 +1,57 @@
+import { join } from 'path';
+import * as os from 'os';
+import * as fs from 'fs';
+import { Chance } from 'chance';
+import * as editJsonFile from 'edit-json-file';
+import { acceptLicenseTerms } from '../../../helpers/database';
+import { MyRedisDatabasePage } from '../../../pageObjects';
+import { commonUrl } from '../../../helpers/conf';
+import { env } from '../../../helpers/constants';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const chance = new Chance();
+
+const workingDirectory = process.env.APP_FOLDER_ABSOLUTE_PATH
+  || (join(os.homedir(), process.env.APP_FOLDER_NAME || '.redisinsight-v2'));
+if (fs.existsSync(workingDirectory)) {
+    const timestampPromoButtonPath = `${workingDirectory}/content/build.json`;
+    const contentPromoButtonPath = `${workingDirectory}/content/create-redis.json`;
+    const timestampPromoButtonFile = editJsonFile(timestampPromoButtonPath);
+    const contentPromoButtonFile = editJsonFile(contentPromoButtonPath);
+    const timestampBeforeUpdate = timestampPromoButtonFile.get('timestamp');
+    const newTimestamp = timestampBeforeUpdate - 1;
+    const newPromoButtonText = chance.word({ length: 10 });
+
+    //Edit json file values
+    contentPromoButtonFile.set('cloud.title', newPromoButtonText);
+    contentPromoButtonFile.set('cloud.description', newPromoButtonText);
+    contentPromoButtonFile.save();
+    timestampPromoButtonFile.set('timestamp', newTimestamp);
+    timestampPromoButtonFile.save();
+
+    fixture `Automatically update information`
```

Comment:
the same fixture name as in previous fixture above

Reply:
changed, thanks